controllers/clusterresources/redisuser_controller.go

```diff
@@ -139,28 +139,28 @@ func (r *RedisUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 				"User has been created for a Redis cluster. Cluster ID: %s, username: %s",
 				clusterID, username)
 
-			finalizerNeeded := controllerutil.AddFinalizer(secret, models.DeletionFinalizer)
+			finalizerNeeded := controllerutil.AddFinalizer(secret, user.GetDeletionFinalizer())
 			if finalizerNeeded {
 				err = r.Update(ctx, secret)
 				if err != nil {
-					l.Error(err, "Cannot update Cassandra user secret",
+					l.Error(err, "Cannot update Redis user secret with deletion finalizer",
 						"secret name", secret.Name,
 						"secret namespace", secret.Namespace)
 					r.EventRecorder.Eventf(user, models.Warning, models.UpdatedEvent,
-						"Cannot assign Cassandra user to a k8s secret. Reason: %v", err)
+						"Cannot assign Redis user to a k8s secret. Reason: %v", err)
```

Comment:
`Cannot assign k8s secret to a Redis user` maybe?

Reply:
Yes, you are right, thank you!
Fixed